sites/friday/src/get-started/flask/_index.md

```diff
@@ -1,4 +1,792 @@
+$ export PLATFORM_PROJECT_ENTROPY=$(openssl rand -base64 32)
+```
+
+And last, source our `.environment` file to finish setting up all the environmental variables in our current
+shell:
+
+```shell
+$ source ./.environment
+```
+
+We now have everything we need for Flask-Migrate to be able to connect to the database and generate our
+migration files. First we need to have Flask-Migrate initiate the migrations directory and prepare for the
+migrate command:
+
+```shell
+$ flask db init
+```
+
+Now we can have Flask-migrate generate our migrations:
+
+```shell
+$ flask db migrate
+```
+
+And now commit our generated migrations:
+
+```shell
+$ git add migrations/*
+$ git commit -m "adds migrations"
+```
+
+We now need to instruct {{% vendor/name %}} to run the Flask-migrate upgrade command when deploying so we know any
+migration changes are automatically applied. Re-open the `{{< vendor/configfile "app" >}}` and find the `deploy` hook
+where we added `npm run build`. On the next line, add `flask db upgrade`.
+
+```yaml {configFile="app"}
+      deploy: |
+        set -eux
+        npm run build
+        flask db upgrade
+```
+
+Commit the changes:
+
+```shell
+$ git add {{< vendor/configfile "app" >}}
+$ git commit -m "adds flask db upgrade to deploy hook"
+```
+
+And finally, push everything up to our {{% vendor/name %}} environment!
+
+```shell
+$ {{% vendor/cli %}} environment:push -y
+```
+
+Congrats! You've now successfully deployed your Flask application to {{% vendor/name %}}! Take a moment to visit your
+site and test it out!
+
+## Local Development
+Now that we have our flask application working on {{% vendor/name %}} we'll need a local environment to be able to make
+quick changes. Luckily flask already has a development server we can take advantage of! And we can utilize
+everything we set up previously with the flask development server.
+
+When we pushed our changes to {{% vendor/name %}} in the last section, it caused our database service to be redeployed
+which closed our tunnel, so first we'll need to reopen it:
+
+```shell
+$ {{% vendor/cli %}} tunnel:open -y
+```
+
+You can verify the tunnel is open and working again by running the following command:
+
+```shell
+$ {{% vendor/cli %}} tunnel:info
+```
+
+Next we'll do the same steps we added to our build hook:
+
+```shell
+$ npm install
+$ npm run build
+```
+
+Now we can run npm start which will run start up webpack watch and start up our flask dev server:
+
+```shell
+$ npm run start
+```
+
+You can now view http://127.0.0.1:5000 in your browser to see your local instance.
+{{< note >}}
+Please note that you are
+connected to your production database so any changes you make locally in the interface (like adding a new
+user) are being added to production. In the next guide will discuss setting up a local development environment
+that creates copies of your database to run locally as well.
+{{< /note >}}
+
+## Switching from the Flask server to an alternative Web Server
```

Comment:
Could we shorten this page by including this as an option/code tab in the `web.commands.start` section?